pkg/vendir/fetch/githubrelease/sync.go

```diff
@@ -165,17 +210,70 @@ func (d Sync) matchesAssetName(name string) (bool, error) {
 	return false, nil
 }
 
+func (d Sync) fetchTagSelection() (string, error) {
+	opt := &github.RepositoryListByOrgOptions{
+		ListOptions: github.ListOptions{PerPage: 10},
```

Comment:
might as well crank it up if we are going to be fetching all of them. how about 50?

Reply:
I adjusted this to 40 so I can continue to use the same repo for the test case. That repo happens to have exactly 50 tags right now, so this way we can fetch from the second page